main.go

```diff
@@ -30,3 +32,23 @@ func helloHandler(w http.ResponseWriter, r *http.Request) {
 	args := mux.Vars(r)
 	fmt.Fprintf(w, "Hello %s!", args["name"])
 }
+
+// initiateTranscriptionJobHandle takes a POST request containing a json object,
+// decodes it into an audioData struct, and returns the struct json-encoded.
+func initiateTranscriptionJobHandler(w http.ResponseWriter, r *http.Request) {
+	var jsonData transcriptionJobData
+
+	// unmarshal from the response body directly into our struct
+	if err := json.NewDecoder(r.Body).Decode(&jsonData); err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
Is this really indicative of an `StatusInternalServerError`? I think it's a client error.